src/TaskModel.vala

```diff
@@ -567,6 +597,10 @@ public class Tasks.TaskModel : Object {
 
         debug (@"Updating task '$(comp.get_uid())' [mod_type=$(mod_type)]");
         yield client.modify_object (comp, mod_type, ECal.OperationFlags.NONE, null);
+
+        if (ECal.util_component_has_x_property (comp, "X-EVOLUTION-GTASKS-POSITION")) {
+            yield update_gtasks_tasklist (list, client, get_registry_sync ());
+        }
```

Comment:
This should be triggered by EDS. From what I understand EDS does trigger the refresh automatically EXCEPT if the `X-EVOLUTION-GTASKS-POSITION` changes on the remote, right?